lib/haml/escapable.rb

```diff
@@ -4,30 +4,31 @@ module Haml
   # Like Temple::Filters::Escapable, but with support for escaping by
   # Haml::Herlpers.html_escape and Haml::Herlpers.escape_once.
   class Escapable < Temple::Filter
```

Comment:
Why do we need our own escapable implementation?

Reply:
That's because Haml has its original way of escaping: `escape_once`. We can't share such logic with any other libraries.